src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -261,6 +261,16 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
         Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId, bool showHistory = false, bool showHistoryOutput = false, bool showInput = true);
 
+        /// <summary>
+        /// Get the status of multiple instances.
+        /// </summary>
+        /// <param name="instanceIds"> The instanceIDs to query.</param>
+        /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="showInput">If set, fetch and return the input for the orchestration instance. If both <c>showHistory</c> and <see cref = "DurableTaskOptions.StoreInputsInOrchestrationHistory" /> are also set to<c>true</c>, then the inputs for activity and sub-orchestration events in the orchestration history will also be returned.</param>
+        /// <returns>Returns a list of orchestration statuses.</returns>
```

Comment:
We should be more descriptive about the order and contents of this list, because there are many different ways the implementation could choose to return the results and we want to make sure customers don't make wrong assumptions. Here is my suggestion:
```suggestion
        /// <returns>
        /// Returns a list of orchestration statuses. The length and order of the returned list will match the length and 
        /// order of <paramref name="instanceIds"/>. If any instance ID doesn't exist, a <c>null</c> value will be set in
        /// the corresponding list element.
        /// </returns>
```